src/js/msp/MSPHelper.js

```diff
@@ -109,6 +109,9 @@ MspHelper.prototype.process_data = function(dataHandler) {
                 $('span.i2c-error').text(CONFIG.i2cError);
                 $('span.cycle-time').text(CONFIG.cycleTime);
                 $('span.cpu-load').text(i18n.getMessage('statusbar_cpu_load', [CONFIG.cpuload]));
+                if (CONFIG.cpuload >= 30) {
+                    GUI.log(i18n.getMessage('cpuLoadTooHigh'));
```

Comment:
Instead of potentially filling the log with these messages, I think we should rather use a symbol in the top bar to indicate a high CPU load.

Reply:
Well, the target audience is newcomers, as experienced users should be familiar with checking the CPU load.  I don't think a symbol is visible enough for that purpose.

We could  differentiate by expert mode enabled so the log is not filled with these messages if other more important log messages are needed for these experienced users.

Reply:
Flooding the log will not really be more visible - newcomers don't even know to look there - and it will be distracting. Making the warning obvious by using text and a red background or something is preferrable.

Reply:
Agreed, should we display the warning if CPU load  >= 90% ? I think that leaves a great enough margin.

Reply:
Also, where do you reckon the text and red background be added? I think a red bar just under the log messages would be best.

Reply:
I'd put it to the left of the 'Motor arming' triangle, underneath the battery indicator.

(Not sure if this triangle is still used, but we could also repurpose it to be active whenever a problem exists, and put text to indicate the reason next to it.)